idx: 450
erasor: 
    max_range: 60.0  # 20 / 15 also good     10/5 For Korea Univ!!
    num_rings: 15    # 35   # 0911_{3, 4} 는 20이었음
    num_sectors: 60 # 108
    min_h: -1.3
    max_h: 3.2
    th_ground_retrieval_h: 0.2 # <- 0.4   height [m]
    th_bin_max_h: 0.05 # [m]
    threshold_h_percentage: 0.3 # percentage: originally, 0.3 yet 0.3 seems like it is quite aggressive!
    minimum_num_pts: 15
    rejection_ratio: 0
    sac_distThr: 0.05
    gf_dist_thr: 0.125
    gf_iter: 3
    gf_num_lpr: 20
    gf_th_seeds_height: 0.4
    version: 3



MapUpdater:
    initial_count: 0
    data_type: "05"
    filename: "/media/shapelim/UX960NVMe1/kitti_semantic/semantic_KITTI_map/src_0_2/seq05/merge_result.pcd" 
    init_registration: false
    env: "outdoor"
    voxel_size: 0.05 
    voxelization_interval: 5
    removal_interval: 4


tf:
     lidar2body: [0.0, 0.0, 1.73, 0, 0.0, 0.0, 1.0] # xyz q_x, q_y, q_z, q_w in order

debug:
    nodegen: true
    registrator: true

# For NodeGen
node:
    dist_diff: 0.2
    angle_diff: 30.0

pc2:
    range_max: 100.0
    range_min: 0.1
    voxel_size: 0.2 # also used in IcpMatcher
    voxelization_interval: 100
    

NDT:
    is_ndt_on: false
    epsilon: 0.01
    step_size: 0.1
    resolution: 2.0
    num_iter_max: 30
    voxel_size: 0.5 
    voxelization_interval: 10
    score: 10.0



